earn/src/components/advanced/BorrowMetrics.tsx

```diff
@@ -236,14 +210,37 @@ export function BorrowMetrics(props: BorrowMetricsProps) {
           <MetricCardPlaceholder height={56} $animate={!userHasNoMarginAccounts} />
           <MetricCardPlaceholder height={56} $animate={!userHasNoMarginAccounts} />
           <MetricCardPlaceholder height={56} $animate={!userHasNoMarginAccounts} />
+          <MetricCardPlaceholder height={56} $animate={!userHasNoMarginAccounts} />
         </MetricsGridLower>
       </MetricsGrid>
     );
 
-  let liquidationDistanceText = '-';
-  if (maxSafeCollateralFall !== null) {
-    if (maxSafeCollateralFall === Number.POSITIVE_INFINITY) liquidationDistanceText = '∞';
-    else liquidationDistanceText = `${(maxSafeCollateralFall * 100).toPrecision(2)}% drop in collateral value`;
+  const etherscanUrl = getEtherscanUrlForChain(activeChain);
+
+  const mostRecentManager = marginAccount.mostRecentModify?.args!['manager'] as Address;
```

Comment:
The logic here contradicts itself. You are using optional chaining and a non-null assertion operator together.

Reply:
Was a result of copy/paste. mostRecentModify can be undefined, but args never should be. Fixed.